### PR TITLE
Readme fix

### PR DESCRIPTION
**Pull request by puneith**
@jerjou @jonparrott Fee free to make modifications. Users are tripping over this section and we need to clarify this better.

### CODE REVIEW

File: speech/api/README.md [hunk 1]
```diff
@@ -48,15 +48,17 @@ Before running these samples perform the steps:
     cd python-docs-samples/speech/api
     ```
 
-* Create virtualenv
+* Create [virtualenv][virtualenv]
```
Comment (jerjou) [inline speech/api/README.md]: Create a [virtualenv][virtualenv]

File: speech/api/README.md [hunk 2]
```diff
@@ -48,15 +48,17 @@ Before running these samples perform the steps:
...
     virtualenv env
     source env/bin/activate
     ```
 
 ### Install the dependencies
 
-The sample uses the [PyAudio][pyaudio] library to stream audio from your computer's microphone.  PyAudio depends on [PortAudio][portaudio], which may need to be compiled when you install PyAudio. If you run into compilation issues that mention PortAudio, you may have to [install some dependencies][pyaudio-install].
+The sample uses the [PyAudio][pyaudio] library to stream audio from your computer's microphone.  PyAudio depends on [PortAudio][portaudio], which may need to be compiled when you install PyAudio. 
+
+* You may run into compilation issues when doing pip install below, in which case [install some dependencies][pyaudio-install].
```
Comment (jerjou) [inline speech/api/README.md]: ...when doing the `pip install` below, in which case you may need to [install some dependencies][pyaudio-install].

**Comment by puneith** [comment]
@jerjou please check. Make the [install some dependencies] absolute.

**Comment by jonparrott** [comment]
LGTM, Jerjou to review and merge.

File: speech/api/README.md [hunk 3]
```diff
@@ -48,15 +48,17 @@ Before running these samples perform the steps:
...
     virtualenv env
     source env/bin/activate
     ```
 
 ### Install the dependencies
 
-The sample uses the [PyAudio][pyaudio] library to stream audio from your computer's microphone.  PyAudio depends on [PortAudio][portaudio], which may need to be compiled when you install PyAudio. If you run into compilation issues that mention PortAudio, you may have to [install some dependencies][pyaudio-install].
+The sample uses the [PyAudio][pyaudio] library to stream audio from your computer's microphone.  PyAudio depends on [PortAudio][portaudio], which may need to be compiled when you install PyAudio. 
+
+* Before proceeding [install some dependencies][pyaudio-install].
```
Comment (jerjou) [inline speech/api/README.md]: Suggest:

The sample uses the [PyAudio][pyaudio] library to stream audio from your computer's microphone.  PyAudio depends on [PortAudio][portaudio], which may need to be installed separately, depending on your platform:

* Install the [pyAudio dependencies][pyaudio-install].

**Comment by puneith** [comment]
@jerjou made changes.

**Pull request merged**